tools/docker/daml-all-in-one/Readme.md

```diff
@@ -0,0 +1,29 @@
+# DAML All in One Image
+
+An all in one DAML docker image with the `sample ledger contracts`.
+
+## Build an image locally
+
+* To build the besu v1.5.1 image locally use:
+  ```sh
+  docker build -t daml-all-in-one .
+  ```
+
+## Running daml-all-in-one container:
+
+```sh
+docker run --privileged -p 6865:6865 -p 7575:7575 daml-all-in-one
+```
+
+```
+The following ports are open on the container:
```

Comment:
The formatting of this block and the subsequent ones seems incorrect.
![image](https://github.com/user-attachments/assets/1adf4cfc-81d0-4558-9738-949e8719d353)

You can install this [VS Code Extension](https://marketplace.visualstudio.com/items?itemName=yzhang.markdown-all-in-one) to preview the markdown files within the editor itself (Preview can be done using Ctrl + Shift + V or Ctrl + K V, once the extension is installed)

Reply:
Hello @jagpreetsinghsasan ,
Review point has been addressed.

Thanks!

Reply:
The commit lint git action is failing @raynatopedrajeta

Reply:
Hello @jagpreetsinghsasan ,
Review point for commit lint has been addressed.

Thank you!